app/Support/ItemCollection.php

```diff
@@ -461,8 +461,9 @@ public function canShootArrows(int $min_level = 1)
                         && ($this->has('Bow') || $this->has('BowAndArrows')));
             case 1:
             default:
-                return ($this->has('Bow') && ($this->has('ShopArrow') || $this->has('SilverArrowUpgrade')))
-                    || $this->has('ProgressiveBow')
+                return (($this->has('Bow') || $this->has('ProgressiveBow')) 
+						&& ($this->has('ShopArrow') || $this->has('SilverArrowUpgrade')))
+                    || $this->has('ProgressiveBow', 2)
```

Comment:
this reads like requiring silver arrows, which is not the default/1 case

Reply:
2x Progressive bow is for Retro mode allowing Silver Arrows as your ammunition.
In the other line i only added progressive bow as alternative to the standard bow.

If I missunderstood the code, I'll be happy to learn.
Will fix the Tab/spaces Part later today.

Reply:
Ok, was able to confirm that the current code makes Silver Arrows required in Non Retro.